src/messages/MOSDOp.h

```diff
@@ -358,11 +358,11 @@ struct ceph_osd_request_head {
 
     // Always keep here the newest version of decoding order/rule
     if (header.version == HEAD_VERSION) {
-	  ::decode(pgid, p);
-	  ::decode(osdmap_epoch, p);
-	  ::decode(flags, p);
-	  ::decode(reassert_version, p);
-	  ::decode(reqid, p);
+      ::decode(pgid, p);
```

Comment:
quote from https://github.com/ceph/ceph/pull/12441#issuecomment-278239985 by @Yan-waller
> Here is just for unified style, and I think it seems to be more suitable that appending these code  to the last "else" branch, since we need only add codes start with "+" as following conveniently if we inc HEAD_VESION in the future version.